libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -189,6 +189,16 @@ AP_GPS_ERB::_parse_gps(void)
             state.time_week_ms    = _buffer.stat.time;
             state.time_week       = _buffer.stat.week;
         }
+        if (   _buffer.ver.ver_high > 0
+           || (_buffer.ver.ver_high == 0 && _buffer.ver.ver_medium >= 2)) {
```

Comment:
Good catch. I was coding for a more general case like version 2.1 vs. version 1.9.
But you are correct it can be simplified in this particular case.